websites/M/MangaWorld/presence.ts

```diff
@@ -24,23 +24,22 @@ presence.on("UpdateData", async () => {
 		presenceData.smallImageKey = "bookmark";
 		presenceData.smallImageText = "Preferiti";
 		presenceData.details = "Sfogliando i preferiti";
-		let filter;
 
-		if (document.location.href.endsWith("RE-READING")) filter = "In rilettura";
-		else if (document.location.href.endsWith("COMPLETED"))
-			filter = "Completati";
-		else if (document.location.href.endsWith("ON_HOLD")) filter = "In pausa";
-		else if (document.location.href.endsWith("PLAN_TO_READ"))
-			filter = "Da leggere";
-		else if (document.location.href.endsWith("DROPPED")) filter = "Droppati";
-		else if (document.location.href.endsWith("READING")) filter = "In lettura";
-
-		presenceData.state = filter;
+		const categories: { [page: string]: string } = {
+				"RE-READING": "In rilettura",
+				COMPLETED: "Completati",
+				ON_HOLD: "In pausa",
+				PLAN_TO_READ: "Da leggere",
+				DROPPED: "Droppati",
+				READING: "In lettura"
+			},
+			[, , , category] = document.location.pathname.split("/");
```

Comment:
Dont use array deconstruction here

Reply:
Just fixed in commit 61a617af5b30425f4ec6837912d80a1765ebe081